exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/DrillRoot.java

```diff
@@ -85,21 +86,33 @@ public ClusterInfo getClusterInfoJSON() {
     // For all other cases the user info need-not or should-not be displayed
     OptionManager optionManager = work.getContext().getOptionManager();
     final boolean isUserLoggedIn = AuthDynamicFeature.isUserLoggedIn(sc);
+    final String processUser = isUserLoggedIn ?
+            ImpersonationUtil.getProcessUserName() : null;
```

Comment:
Do we want to display the process user only if the user is logged in? Here, are we assuming logged in means authenticated? Why not display the process user otherwise?

Rather than null, should we just use a tag such as "<process user>"?

Reply:
From security perspective it would be good to display process user only if an admin user is logged in. Removed the check here as the condition is handled in html.